src/cf-tables/src/cf-table-row-links.js

```diff
@@ -0,0 +1,54 @@
+
+/* ==========================================================================
+   Table Row Links
+
+   Mixin for adding row link click functionality to table organism.
+
+   ========================================================================== */
+
+'use strict';
+
+var closest = require( 'atomic-component/src/utilities/dom-closest' ).closest;
+
+var TableRowLinks = {
+
+  events: {
+    'click tbody tr': 'onRowLinkClick'
+  },
+
+  ui: {
+    base: '.o-table__row-links',
```

Comment:
The comma on the end of this line shouldn't be necessary. But I really like this pattern.